cmd/sonobuoy/app/logs.go

```diff
@@ -18,18 +18,23 @@ package app
 
 import (
 	"fmt"
+	"io"
 	"os"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	ops "github.com/heptio/sonobuoy/pkg/client"
+	"github.com/heptio/sonobuoy/pkg/client"
 	"github.com/heptio/sonobuoy/pkg/errlog"
 )
 
-var logConfig ops.LogConfig
+var logConfig client.LogConfig
 var logsKubecfg Kubeconfig
 
+const (
+	bufferSize = 4096
```

Comment:
why const this twice, you could export from logs and reference that one.

Reply:
they can be independently set, they just happen to be the same. I think exporting & referencing might cause confusion later on "why are these the same reference? do the buffers have to be the same size?"